cmd/scollector/collectors/iterable.go

```diff
@@ -0,0 +1,95 @@
+package collectors
+
+import (
+	"context"
+	"time"
+
+	"bosun.org/cmd/scollector/conf"
+	"bosun.org/metadata"
+	"bosun.org/opentsdb"
+	"bosun.org/slog"
+	"github.com/bosun-monitor/statusio"
+)
+
+// iterable.io collector
+// to use add this in your scollector.toml:
+// [Iterable]
+//   StatusBaseAddr = "https://iterable.statuspage.io"
+//   # TsdbPrefix = "iterable.status."
+//   # MaxDuration = 3 # seconds
```

Comment:
if we keep this feature, I think is should be a duration. Example of how this is done in Bosun with this type https://github.com/bosun-monitor/bosun/blob/master/cmd/bosun/conf/system.go#L560

Reply:
I can do that if needed. We may not want to put too much efforts into it though, if we have sensible defaults.

Reply:
If we are keeping the the option I would like it to be a duration for consistency across Bosun and collector config (where possible). 

If we just take a default and don't make an option in the toml file, then yes, don't really care that much about the type.